src/main/java/teammates/common/datatransfer/FeedbackSessionAttributes.java

```diff
@@ -316,6 +316,13 @@ public boolean isClosingWithinTimeLimit(int hours) {
                && differenceBetweenDeadlineAndNow >= hours - 1
                && differenceBetweenDeadlineAndNow < hours;
     }
+    
+    /**
+     * returns true if the session is closed within the past hour of calling this function
+     */
+    public boolean isClosedWithinPastHour() {
+        return TimeHelper.isClosedWithinPastHour(endTime, gracePeriod, timeZone);
```

Comment:
This is the incorrect abstraction. `TimeHelper` should tell you if time x is within one hour of time y, but it should not know anything about sessions or deadlines.
```java
return TimeHelper.isWithinPastHour(deadline_after_factoring_in_time_zone_and_grade_period)
```